serverless-test-project/serverless.yml

```diff
@@ -86,6 +95,14 @@ functions:
           method: post
           path: "subscription"
 
+  eventsRule:
+    handler: rule-handler.handleRule
+    events:
+      - eventBridge:
+          pattern:
+            detail-type: 
+            - Invoke Lambda Function
```

Comment:
If you want to test invocation failures, you might want to change the retry behaviour so Lambda does not continually retry the failed event. See https://www.serverless.com/framework/docs/providers/aws/events/event-bridge#adding-a-retry-policy-to-an-event-rule